lib/wda/iproxy.js

```diff
@@ -30,7 +31,12 @@ class iProxy {
         connection.destroy();
       }
     });
+    const status = new B((resolve, reject) => {
```

Comment:
should we also reject this promise if an error happens in one of the above callbacks for connection and socket?

Reply:
That shouldn't be necessary. Those callbacks are used when the port is available